ignite/utils.py

```diff
@@ -169,3 +171,32 @@ def manual_seed(seed: int) -> None:
         np.random.seed(seed)
     except ImportError:
         pass
+
+
+def deprecated(deprecated_in: str, removed_in: str = "", reasons: list = [], raise_exception: bool = False) -> Callable:
+
+    F = TypeVar("F", bound=Callable[..., Any])
+
+    def decorator(func: F) -> F:
+        func_doc = func.__doc__ if func.__doc__ else ""
+        deprecation_warning = (
+            f"This function has been deprecated since version `{deprecated_in}`"
+            + (f" and will be removed in version `{removed_in}`" if removed_in else "")
+            + ".\n Please refer to the documentation for more details."
+        )
+
+        @functools.wraps(func)
+        def wrapper(*args: int, **kwargs: float) -> Callable:
```

Comment:
shouldn't it be `def wrapper(*args: Any, **kwargs: Any) -> Callable:` ?

Reply:
I can't find the `mypy` docs link for this, but this [link](https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values) is related. (As in I referred to this while deciding the types)

Reply:
@Devanshu24 previously we annotated args and kwargs as Any as function's arg and kwargs can be `Tuple[Any]` and `Dict[str, Any]`. Let's keep it with Any. 

In the mypy docs they give an example where they have ints for args and float for kwargs.

Reply:
Okay, I wasn't able to get that. I will change it to `Any`. And also read up a little more on type annotations to get a better understanding.

Reply:
we used mainly `Any` and `Any` for both, but I also like `Dict[str, Any]` for kwargs.